lib/spack/docs/basic_usage.rst

```diff
@@ -151,8 +150,8 @@ to get increasingly detailed output.
 The last line, with the ``[+]``, indicates where the package is
 installed.
 
-Add the debug option -- ``spack  install -d mpileaks`` -- to get additional
-output.
+Add the Spack debug option (one or more times) -- ``spack -d install
+mpileaks`` -- to get additional (and even more verbose) output.
```

Comment:
Does each additional `-d` added increase how verbose the output is?

Reply:
Up to 3

Reply:
Or you can simply use `-ddd` for full debug output.